### PR TITLE
{Aladdin} add usage examples for module: keyvault monitor network privatedns resource storage vm 

### PR DESCRIPTION
**Pull request by azclibot**
This PR is created by AladdinParser Pipeline. 

 ADO_BUILD_ID=660624 

 Against last commit of dev: https://github.com/Azure/azure-cli/commit/57a0b8101 

 Against last commit of Aladdin-src: https://github.com/Azure/azure-cli/commit/297c6f4be1cef5ae534efff36ddd397d9e37a150

### CODE REVIEW

**Comment by yonzhan** [comment]
Aladdin

File: src/azure-cli/azure/cli/command_modules/network/_help.py [hunk 1]
```diff
@@ -352,6 +352,10 @@
     text: |
         az network application-gateway http-listener update -g MyResourceGroup --gateway-name MyAppGateway \\
             -n MyHttpListener --host-name www.mynewhost.com
+  - name: Update an HTTP listener. (autogenerated)
+    text: |
+        az network application-gateway http-listener update --gateway-name MyAppGateway --host-names www.mynewhost.com --name MyHttpListener --resource-group MyResourceGroup
+    crafted: true
```
Comment (kairu-ms) [inline src/azure-cli/azure/cli/command_modules/network/_help.py:358]: Remove this example. It's the same as upper one.

File: src/azure-cli/azure/cli/command_modules/network/_help.py [hunk 2]
```diff
@@ -2045,6 +2066,10 @@
 examples:
   - name: Get the details of an NS record set.
     text: az network dns record-set ns show -g MyResourceGroup -z www.mysite.com -n MyRecordSet
+  - name: Get the details of an NS record set. (autogenerated)
+    text: |
+        az network dns record-set ns show --name MyRecordSet --resource-group MyResourceGroup --subscription mysubscription --zone-name www.mysite.com
+    crafted: true
```
Comment (kairu-ms) [inline src/azure-cli/azure/cli/command_modules/network/_help.py:2072]: Remove this one. It's the same as upper one.

File: src/azure-cli/azure/cli/command_modules/network/_help.py [hunk 3]
```diff
@@ -2448,6 +2481,10 @@
   - name: Delete a link authorization of an ExpressRoute circuit.
     text: >
         az network express-route auth delete --circuit-name MyCircuit -g MyResourceGroup -n MyAuthorization
+  - name: Delete a link authorization of an ExpressRoute circuit. (autogenerated)
+    text: |
+        az network express-route auth delete --circuit-name MyCircuit --name MyAuthorization --resource-group MyResourceGroup --subscription mysubscription
+    crafted: true
```
Comment (kairu-ms) [inline src/azure-cli/azure/cli/command_modules/network/_help.py:2487]: Remove this one. It's the same as upper one.

**Review by kairu-ms — CHANGES_REQUESTED** [review]

**Pull request closed without merging**